packages/components/addon/components/hds/dropdown/list-item.hbs

```diff
@@ -25,7 +43,7 @@
       >
         {{#if @icon}}
           <div class="hds-dropdown-list-item__interactive-icon">
-            <FlightIcon @name={{@icon}} />
+            <FlightIcon @name={{@icon}} @isInlineBlock={{false}} />
```

Comment:
[issue] the same change needs to be done at lines `57` and `68`

PS: see why long time ago I told you that I was worried that non abstracting these replicated code would lead to these errors? I suggested using handlebars partials, do you think this would be possible?